portfolio/src/main/webapp/script.js

```diff
@@ -27,8 +27,8 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-/**
- * Adds a random fact about me to the page.
+/*
+ * Add a random fact about me to the about section of main page.
  */
 function addRandomFact() {
```

Comment:
I'm going to get super nitpicky, every time you click this button you want a different fact. An easy way to manage this would be to add a little state.

```
let randomFact = 0;
const factoids = [ ... ];

function addRandomFact() {
  // Get a different index.
  const index = randomFact + Math.random() * (factoids.length - 1);
  // Make sure the index doesn't overflow.
  const factoid = factoids[index % factoids.length];
  ...
}
```

Reply:
Okay, I think I implemented it so that a different fact appears each time? I used your code but didn't copy it exactly.